package.json

```diff
@@ -21,7 +21,11 @@
   },
   "bugs": "https://github.com/jquery/jquery-color/issues",
   "license": "MIT",
+  "main": "jquery.color.js",
   "dependencies": {},
+  "peerDependencies": {
+    "jquery": ">=1.5.0"
```

Comment:
Please change to `>=1.11.0`. Older versions of this package on `npm` are Node-only shims using `contextify` from before we took ownership of the `jquery` package, they should not be used and they will not "just work" with this plugin.